lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -240,6 +243,30 @@
                         });
                         this.setActiveCourseRun(selectedCourseRun);
                     }
+                },
+
+                // update marketing url for deep linking if is_mobile_only true
+                updateMarketingUrl: function(courseRun) {
+                    if (courseRun.is_mobile_only === true) {
+                        var marketingUrl = courseRun.marketing_url, // eslint-disable-line vars-on-top
+                            href = marketingUrl,
+                            path,
+                            start;
+
+                        if (marketingUrl.indexOf('course_info?path_id') < 0) {
+                            start = marketingUrl.indexOf('course/');
+
```

Comment:
Are we sure that url structure will not change.

Reply:
@ahsan-ul-haq Yes.